94ffe8f1841a5bea6566bdb0f88edf344ecd63a7 M4: tamper SNAPSHOT bytes field only

 .ufo/waves/round-31/sources/SNAPSHOT.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
commit 94ffe8f1841a5bea6566bdb0f88edf344ecd63a7
Author: mut <mut@local>
Date:   Fri Oct 23 20:07:42 2026 -0300

    M4: tamper SNAPSHOT bytes field only

diff --git a/.ufo/waves/round-31/sources/SNAPSHOT.json b/.ufo/waves/round-31/sources/SNAPSHOT.json
index a1173d8..b795829 100644
--- a/.ufo/waves/round-31/sources/SNAPSHOT.json
+++ b/.ufo/waves/round-31/sources/SNAPSHOT.json
@@ -7,7 +7,7 @@
   "server-common.cpp": {
    "path": "tools/server/server-common.cpp",
    "sha256": "b4f4aae7eed919f914d132ae64416a0e7cfb23442161af6f63ba8d902954d442",
-   "bytes": 75061
+   "bytes": 1
   },
   "server-schema.cpp": {
    "path": "tools/server/server-schema.cpp",
